drivers/gpio/gpio_sx1509b.c

```diff
@@ -436,9 +430,30 @@ static int sx1509b_init(struct device *dev)
 
 	k_sleep(K_MSEC(RESET_DELAY_MS));
 
+	/* Reset state */
```

Comment:
I guess this comment referred to the reset state of the SX1509B, so it does not seem to apply any longer.